.github/workflows/pr-dev.yml

```diff
@@ -25,14 +26,6 @@ jobs:
     - name: Unit tests
       run: go test -short -v ./...
 
-    - name: Integration tests

```

Comment:
Is it supposed to remove this feature or should it be added to workflows-microservice project?